css/droplet.css

```diff
@@ -263,7 +262,7 @@
   max-height: 150px;
   position: absolute;
   background-color: #FFF;
-  z-index: 10000;
+  z-index: 500;
```

Comment:
This matches our enforced value from https://github.com/code-dot-org/code-dot-org/blob/1e3ae0c714352eb2341c0b5b548009bc98c74283/apps/style/applab/style.scss#L489
z-index of 10000 was causing problems as it was showing up in front of dialogs. We forcibly set the z-index to 500 with !important. Setting to 500 will make this more usable for other users of the droplet repo as well.